src/main.rs

```diff
@@ -886,40 +968,51 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         }
     }
 
-    for client in cluster_images.get_clients_mut() {
-        let client_index = if let ValidatorType::Client(index) = client.validator_type() {
+    if let ClientConfig::Generic(ref generic_config) = client_config {
+        info!(
+            "Waiting {}s before deploying client",
+            generic_config.delay_start
+        );
+        std::thread::sleep(std::time::Duration::from_secs(generic_config.delay_start));
+    }
```

Comment:
Would there be a negative impact on the system if we omitted the bench-tps wait argument and had all clients (including bench-tps) do their waiting over here?

Reply:
i actually think the bench-tps `client_wait_for_n_nodes` is better than `delay_start`. `client_wait_for_n_nodes` is actually translated into `--num-nodes` in `bench-tps`, so it's built into the `bench-tps` code. the client will wait until there are n nodes caught up and voting in the network before launching transactions. It's a more precise wait flag than `delay_start`.  `delay_start` is just an arbitrary value that you can pass in so client only starts sending txs/spam after a certain amount of time (aka like waiting some expected time for clients to catch up). 

Open to your thoughts though! but that's just why they are currently separate flags slash implemented in different areas

Reply:
Gotcha, sounds like the current approach is best then, thanks for the explanation